src/AuthModule/Components/ResetPassword/ResetPassword.tsx

```diff
@@ -1,7 +1,172 @@
-import React from 'react'
+import React, { useState } from 'react'
+import Loader from '../../../SharedMoudule/Components/Loading/Loading'
+import { useNavigate } from 'react-router-dom';
+import logo from "../../../assets/PMS 3.svg";
+import { useForm } from 'react-hook-form';
+import { OTPValidation, emailValidation, passwordValidation } from "../Validator/Validator.js";
+import { baseUrl } from '../../../Constants/Components/Urls';
+import { toast } from 'react-toastify';
+import axios from 'axios';
 
 export default function ResetPassword() {
+
+  const navigate=useNavigate();
+  // ########################## submit New password ##################################
+
+  const [isLoading,setIsLoading]=useState(false);
+  const {register,handleSubmit,formState:{errors}}=useForm();
+
+  const submitNewPassword=async(data:any)=>{
+    try {
+      setIsLoading(true);
+      const response = await axios.post(`${baseUrl}/Users/Reset`,data);
+        console.log(response);
+        toast.success(" reset New password please!")
+      
+        navigate("/")
+    
+    } catch (error) {
+      console.log(error)
+      toast.error(error?.response?.data?.message);
+      setIsLoading(false);
+    }
+    }
+// ########################## show password ##################################
+
+    const[showPassword,setShowPassword]=useState(true);
```

Comment:
Currently, we have one state for two password inputs, when we click the toggle button, it will toggle both password inputs. We need a state for each one so that when someone click the toggle button, it will only toggle the corresponding input.

Here is an illustration of what will happen:
![toggle password](https://github.com/SalmaAboArab/Project-Management-System/assets/64327685/1748b510-f745-4750-a4f8-18b15d4a6197)

It is from the change Password page, but the same idea applies.